packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -329,3 +334,29 @@ export function convertSummaryTreeToITree(summaryTree: ISummaryTree): ITree {
         unreferenced: summaryTree.unreferenced,
     };
 }
+
+export class TelemetryContext implements ITelemetryContext {
+    private telemetry = new Map<string, any>();
```

Comment:
We do not allow 'any' in telemetry. So I'd limit it across the stack to types that we support in telemetry, or maybe go even more restrictive and say we support numbers, maybe strings.